scripts/find-links-localize.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+#
+# Lists the `en` intra-site links on a localized Markdown page.
+# Checks if a localized page exists. Reports results.
+# Next step: modify the links on the page by prefixing the link with the <lang> and add/commit changes.
+# If there is no localized version of the page, validate the `en` link. Report results.
+# A temporary results file is created which lists the localized pages and the validated links.
+# To generate the file, run the script from the website/script directory with a <lang> parameter.
+#
+# Usage: python3 find-links-localize.py zh
+
+import argparse
+import glob
+import os
+import platform
+import re
+import sys
+import tempfile
+from pathlib import Path
+
+DIRS = ["concepts", "contribute", "home", "reference", "setup", "tasks", "tutorials"]
+TEST_DIRS = ["concepts"]
+tmp_file_output = {}
+
+def write_to_tmp():
+    """Write validation results to a temporary file."""
```

Comment:
Will we report this in the CI?

Reply:
Hi @irvifa . Good questions. Thanks for your feedback.
I have not considered running this script as part of CI. This script is really an initial experimentation.